cartesi-rollups_versioned_docs/version-1.5/external-resources/community-tools.md

```diff
@@ -0,0 +1,173 @@
+---
+id: community-tools
+title: Community tools
+---
+
+Several tools created and maintained by the community streamline the dApp creation process on Cartesi Rollups.
+
+## Deroll
+
+Introducing Deroll, a powerful TypeScript framework designed to simplify the development of dApps on Cartesi.
+
+- **Features**:
+
+  - Simplifies dApp development with intuitive methods.
+  - Handles advance and inspect requests easily.
+  - Comprehensive wallet functionality for ERC20, ERC721 and ERC-1155 token standards.
+  - Integrated router for complex routing logic.
+
+- **Getting Started**:
+
+  - Create a new Deroll project by running:
+    ```bash
+    npm init @deroll/app
+    ```
+
+- **Resources**:
+  - [Deroll Documentation](https://deroll.dev)
+  - [Deroll GitHub Repository](https://github.com/tuler/deroll)
+
+---
+
+## NoNodo
+
+NoNodo is a cutting-edge development tool for Cartesi Rollups that allows applications to run directly on the host machine, bypassing Docker or RISC-V compilation.
+
+- **Features**:
+
+  - Run applications directly on the host machine for faster performance.
```

Comment:
Should include a disclaimer that this is a non-reproducible execution mode, which is not provable on-chain. It is a tool for providing a faster iterative development experience, and cannot be used by deployed applications.